plugins/woocommerce/includes/class-wc-mobile-messaging-handler.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Prepares formatted mobile deep link navigation link for order mails
+ *
+ * @package WooCommerce\Emails
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Mobile_Messaging_Handler
+ */
+class WC_Mobile_Messaging_Handler {
+
+	const OPEN_ORDER_INTERVAL_DAYS = 30;
+
+	/**
+	 * Prepares mobile messaging with a deep link
+	 *
+	 * @param WC_Order $order order that mobile message is created for.
+	 * @param ?int     $blog_id  of blog to make a deep link for.
```

Comment:
I was initially just a little confused by [line 35](https://github.com/woocommerce/woocommerce/pull/34467/files#diff-198890d4431c1f21d2d0a9d3ee3b9d8846cd83ec80bf2b028032459158ff7b1aR35):

```
$has_jetpack            = null !== $blog_id;
```

As it made me wonder what the blog ID has to do with whether Jetpack is present. Now I've read `WC_Email::mobile_messaging()` it's a little clearer, but maybe we could also make a small amendment here:

```suggestion
	 * @param ?int     $blog_id  of blog to make a deep link for (will be null if Jetpack is not enabled).
```

Reply:
Yes, that was indeed confusing. Thanks, I've added the suggested clarification.